common/src/service/CommonConstants.cs

```diff
@@ -118,8 +127,11 @@ static class CommonConstants
 
         // Custom HTTP headers
         public const string IotHubErrorCode = "IotHubErrorCode";
+
         public const string HttpErrorCodeName = "iothub-errorcode";
 
+        public const string IotHubAadTokenScope = "https://iothubs.azure.net/.default";
```

Comment:
Does .NET allow you to declare a const string[] here rather than creating a new string array each time the token is retrieved?

Reply:
It does. I was just torn between that or this, went back and forth and finally chose the current implementation thinking it might be more readable to look at the implementation. But I do see your point and think making a const string[] might be better.

Reply:
FWIW, I would not choose the one that required array construction each time. `static readonly` is sufficiently comparable to `const`. I don't see a downside there.